scripts/provisioning/s3_haproxy_config.py

```diff
@@ -240,3 +247,14 @@ def process(self):
     os.system("rm -rf /etc/cron.daily/logrotate")
     os.system("cp /opt/seagate/cortx/s3/install/haproxy/logrotate/logrotate /etc/cron.hourly/logrotate")
     os.system("systemctl restart rsyslog")
+
+  def set_haproxy_restart_to_no(self):
```

Comment:
As haproxy is 3rd party its service file should be managed by provisioner and not in component config, @ujjwalpl  please suggest.

Reply:
We have discussed with @SomwanshiAnjali she is going to update on the provisioner side